addon/ng2/blueprints/ng2/files/karma-test-shim.js

```diff
@@ -0,0 +1,44 @@
+Error.stackTraceLimit = Infinity;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000;
+
+__karma__.loaded = function() {};
+
+System.config({
+  packages: {
+    'base/dist/app': {
+      defaultExtension: false,
+      format: 'register',
+      map: Object.keys(window.__karma__.files)
+        .filter(onlyAppFiles)
+        .reduce(function(pathsMapping, appPath) {
+          var moduleName = appPath.replace(/^\/base\/dist\/app\//, './').replace(/\.js$/, '');
+          pathsMapping[moduleName] = appPath + '?' + window.__karma__.files[appPath]
+        return pathsMapping;
+      }, {})
+    }
+  }
+});
+
+System.import('angular2/src/core/dom/browser_adapter').then(function(browser_adapter) {
```

Comment:
can you add a todo comment that once beta is out we should change this code to use a "test platform".